frame.js

```diff
@@ -0,0 +1,56 @@
+class Frame {
+  constructor(tenth = false) {
+    this.frame = []
+    this.tenth = tenth
+  }
+
+  roll(ball) {
+    this.frame.push(ball)
+  }
+
+  score() {
+    return this.frame.reduce((a, b) => a + b, 0);
+  }
+
+  done() {
```

Comment:
could possibly change the name of 'done' and 'donetenth' but the code for frame is very clear :)!